src/overtone/sc/synth.clj

```diff
@@ -670,9 +670,12 @@
     (with-meta
       (map->Synth
        {:name s-name
-        :sdef sdef})
-      (merge {:overtone.helpers.lib/to-string #(str (name (:type %)) ":" (:name %))}
-             (meta s-name)))))
+        :sdef sdef
+        :args params})
+      (merge {:overtone.live/to-string #(str (name (:type %)) ":" (:name %))}
+             (-> (meta s-name)
+                 ;; Eval so we can get rid of the extra quote.
+                 (update :arglists eval))))))
```

Comment:
Is there a way to avoid `eval`?

Reply:
Yeah, used `rest` + `first`, let me know if you know a better way to `unquote` something, thanks!

Reply:
Hmm here we've moved `synth-form` from compile time to runtime, probably the cleanest solution would be to have a runtime equivalent to `synth-form`.

Reply:
Are we also redundantly quoting `params` in the same way?

Reply:
Yeah, it's used only here, so I've thought we wouldn't needed it, but I've just added a `compile-time?` flag to it